xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -1555,6 +1555,71 @@ JSONRPC_STATUS CPlayerOperations::GetPropertyValue(PlayerType player, const std:
         break;
     }
   }
+  else if (property == "currentvideostream")
+  {
+    switch (player)
+    {
+    case Video:
+      if (g_application.m_pPlayer->HasPlayer())
+      {
+        result = CVariant(CVariant::VariantTypeObject);
+        int index = g_application.m_pPlayer->GetVideoStream();
+        if (index >= 0)
+        {
+          SPlayerVideoStreamInfo info;
+          g_application.m_pPlayer->GetVideoStreamInfo(index, info);
+
+          result["index"] = index;
+          result["name"] = info.name;
+          result["language"] = info.language;
+          result["codec"] = info.videoCodecName;
+          result["width"] = info.width;
+          result["height"] = info.height;
+        }
+      }
+      else
+        result = CVariant(CVariant::VariantTypeNull);
+      break;
+
+    case Audio:
+    case Picture:
+    default:
+      result = CVariant(CVariant::VariantTypeNull);
+      break;
+    }
+  }
+  else if (property == "videostreams")
+  {
+    result = CVariant(CVariant::VariantTypeArray);
+    switch (player)
+    {
+    case Video:
+      if (g_application.m_pPlayer->HasPlayer())
```

Comment:
no guarantee that after this returned true there is still a player. you need to rely on the return values only